Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.cs

```diff
@@ -215,15 +218,47 @@ private void OnImageFailed(object sender, ExceptionRoutedEventArgs e)
             VisualStateManager.GoToState(this, FailedState, true);
         }
 
-        private void ImageExBase_EffectiveViewportChanged(FrameworkElement sender, EffectiveViewportChangedEventArgs args)
+        private void ImageExBase_LayoutUpdated(object sender, object e)
         {
-            var bringIntoViewDistanceX = args.BringIntoViewDistanceX;
-            var bringIntoViewDistanceY = args.BringIntoViewDistanceY;
+            InvalidateLazyLoading();
```

Comment:
We should be explicitly checking the `EnableLazyLoading` property here before we call the method (or do this check at the top of the method), eh? As LayoutUpdated is registered on the control it's going to be called in both scenarios?

Reply:
@h82258652 thoughts? ⬆️

Reply:
@michael-hawker @Kyaa-dost Thanks for the suggestion. I updated the code, only register ```LayoutUpdated``` while ```EnableLazyLoading``` is ```true```.